openpower/package/alternate-gcc/alternate-gcc.mk

```diff
@@ -0,0 +1,65 @@
+################################################################################
+#
+# alternate-gcc
+#
+################################################################################
+
+#
+# Based on buildroot/package/gcc/*.mk, but trying to simplify since we're not
+# (yet) going for a general scenario here
+#
+
+ALTERNATE_GCC_VERSION = $(call qstrip,$(BR2_ALTERNATE_GCC_VERSION))
+ALTERNATE_GCC_SITE = $(BR2_GNU_MIRROR:/=)/gcc/gcc-$(ALTERNATE_GCC_VERSION)
+ALTERNATE_GCC_SOURCE = gcc-$(ALTERNATE_GCC_VERSION).tar.xz
+
+HOST_ALTERNATE_GCC_SUBDIR = build
+
+HOST_ALTERNATE_GCC_DEPENDENCIES = \
+	host-alternate-binutils \
+	host-gmp \
+	host-mpc \
+	host-mpfr \
+	$(BR_LIBC)
+
+HOST_ALTERNATE_GCC_EXCLUDES = \
+	libjava/* libgo/*
+
+define HOST_ALTERNATE_GCC_CONFIGURE_SYMLINK
+	mkdir -p $(@D)/build
+	ln -sf ../configure $(@D)/build/configure
+endef
+
+HOST_ALTERNATE_GCC_CONF_OPTS += \
+	$(call qstrip,$(BR2_ALTERNATE_GCC_EXTRA_CONFIG_OPTIONS))
+
+define  HOST_ALTERNATE_GCC_CONFIGURE_CMDS
```

Comment:
assuming this is all the same

Reply:
I tried to make it as close to buildroot's as "necessary" - problem is that they are pretty spread out, and of course, possibly changing with each buildroot release.. see references at https://github.com/open-power/buildroot/blob/a84aaaeacbd1a69053cde2e50f26250e9328de1e/package/gcc/gcc.mk#L73 and https://github.com/open-power/buildroot/blob/a84aaaeacbd1a69053cde2e50f26250e9328de1e/package/gcc/gcc-final/gcc-final.mk#L62

We can probably make the conf command completely static and mimic the buildroot's command verbatin.. if we think this is necessary

Reply:
there's also the possibility of adjusting per-config options using `BR2_ALTERNATE_GCC_EXTRA_CONFIG_OPTIONS`